### DISCUSSION TITLE
在图片前加入路径时，没有拼接路径

### DISCUSSION THREAD

**Discussion by hooye** [Molunerfinn/PicGo#821] [Ideas]
我会在图片前加上路径以作为目录，但picgo 不会拼接路径
图片 a.png b/a.png 路径仍为 URL/a.png ,不会为 URL/b/a.ping
这样上传是成功的，但是需要手动修改，麻烦修复一下

**Comment by Molunerfinn** [comment]
可以找插件处理，有类似功能的插件
